README.md

```diff
@@ -40,17 +36,18 @@ const config = {
 On your `.storybook/preview.ts` file, add the following:
 
 ```js
-...
-import MyComponentsLibrary from 'my-components-library';
-import MyIconsLibrary from 'my-icons-library';
-...
+import MyComponentsLibrary from "my-components-library";
+import MyIconsLibrary from "my-icons-library";
+import { generateAutocompletions } from "storybook-addon-playground";
+const documentation = reactDocgen.parse("../path/to/your/components");
```

Comment:
Where is `reactDocgen` imported from? Is it a peer-dep?

Reply:
a dev dep
```package.json
"react-docgen": "^7.0.3",
```

Reply:
Yes but would the users of the library have to install reactDogen?

Reply:
Because according to the docs they need to use `reactDocgen.parse`

Reply:
yep, they should, but they can have it as a dev dep or run @react-docgen through their cli and save it to a json file
but they do not have to use autocompletions, so it feels harsh adding it as a peer
I'm only using it on the addon for TS purposes

Reply:
So maybe add the import to the example so it would be clear? It's not imported here

Reply:
instead of the parse example here?

Reply:
lmk what you think now 🤞🏼